.github/workflows/int_test.yml

```diff
@@ -7,8 +7,8 @@ on:
         type: string
         required: true
         default: ops-cli-int-test-rg
-      template-content:
-        description: Custom AIO deployment template file contents, used for all deployments in this workflow.
+      runtime-init-args:
+        description: Additional init arguments (beyond cluster name, resource group, key vault, and service principal arguments).
         type: string
         required: false
```

Comment:
do we also need `default: ''` here? In case they don't provide it in `workflow_call`, will it get resolved as null or `''`?

Reply:
I don't want to take any chances and just say '' since null and '' are treated differently

Reply:
Have you tested `workflow_call`? The default is set to `''` [here](https://github.com/Azure/azure-iot-ops-cli-extension/pull/254/files#diff-3034c5f172c4708ce5a8534f97d8430c2052be88efba38cb968b470b73a7349dR52) in `workflow_dispatch`, I figured this way it would run the same regardless of how the test was triggered.

Reply:
oh whoops - will update the workflow call - I do not want to take chances